tests/drivers/gpio/gpio_basic_api/boards/nrf54lm20dk_nrf54lm20a_cpuflpr.overlay

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "nrf54lm20dk_nrf54lm20a_common.overlay"
```

Comment:
Note we will have a A and B SOC variant, but for these overlays there won't be a difference. So it may be better to call it `nrf54lm20dk_nrf54lm20_common.overlay`